lib/cli/commands/placeorder.ts

```diff
@@ -2,29 +2,33 @@ import { callback, loadXudClient } from '../command';
 import { Arguments } from 'yargs';
 import { PlaceOrderRequest, Order } from '../../proto/xudrpc_pb';
 
-export const command = 'placeorder <pair_id> <quantity> [price]';
+export const command = 'placeorder <pair_id> <order_id> <price> [quantity]';
```

Comment:
Was it intentional to make `price` required and `quantity` optional? I think we should keep this the other way around, `<order_id> <quantity> [price]`.